src/sandboxes/web3-react-v6-sandbox/App.tsx

```diff
@@ -0,0 +1,238 @@
+/**
+ * @DEV: If the sandbox is throwing dependency errors, chances are you need to clear your browser history.
+ * This will trigger a re-install of the dependencies in the sandbox – which should fix things right up.
+ * Alternatively, you can fork this sandbox to refresh the dependencies manually.
+ */
+import React, { useState, useCallback } from 'react';
+import styled from 'styled-components';
+
+import { TLog } from './types';
+import { Logs, Sidebar } from './components';
+
+import { useWeb3React, Web3ReactProvider } from '@web3-react/core';
+import { Web3Provider } from '@ethersproject/providers';
+import { PhantomConnector } from "web3-react-v6-phantom"
+import { Signer } from 'ethers';
+import { parseUnits } from 'ethers/lib/utils';
+
+const phantom = new PhantomConnector({
+  supportedChainIds: [1, 5],
+});
+
+// =============================================================================
+// Styled Components
+// =============================================================================
+
+const StyledApp = styled.div`
+  display: flex;
+  flex-direction: row;
+  height: 100vh;
+  @media (max-width: 768px) {
+    flex-direction: column;
+  }
+`;
+
+// =============================================================================
+// Constants
+// =============================================================================
+
+const message = 'To avoid digital dognappers, sign below to authenticate with CryptoCorgis.';
+const TX = {
+  to: '0x0000000000000000000000000000000000000000',
+  value: parseUnits('1', 'wei'),
+  chainId: 0x5, // Goerli network
+};
+// =============================================================================
+// Typedefs
+// =============================================================================
+
+export type SidebarMethods =
+  | {
+      name: string;
+      onClick: () => Promise<string>;
+    }
+  | {
+      name: string;
+      onClick: () => Promise<void>;
+    }
+  | {
+      name: string;
+      onClick: () => void;
+    };
+
+interface Props {
+  // connectedMethods: ConnectedMethods[];
+  logs: TLog[];
+  clearLogs: () => void;
+  createLog: (log: TLog) => void;
+}
+
+// =============================================================================
+// Hooks
+// =============================================================================
+
+/**
+ * @DEVELOPERS
+ * The fun stuff!
+ */
```

Comment:
Can probably remove this comment since this is carry over from the multichain sandbox. For this sandbox, all the functionality is now in the components and no longer in the hooks